Property files added

PavelAnhur opened the pull request.

PavelAnhur committed:
builder for Google cloud form and properties files

PavelAnhur committed:
builder for Google cloud form and properties files

dzmitry-panimash commented on target/classes/env1.properties:
```diff
@@ -0,0 +1,6 @@
+browser=Chrome
```
please, update .gitignore - we don't need the target folder in git at all

dzmitry-panimash commented on target/classes/env2.properties:
```diff
@@ -0,0 +1 @@
+browser=Firefox;
```
the same here

dzmitry-panimash commented on src/test/java/webdriver/WebDriverSetup.java:
```diff
@@ -20,7 +21,7 @@ public static WebDriver getDriver() {
 
 	private static void setupWebDriver() {
-		switch (System.getenv("WebDriver")) {
+		switch (new PropertyValue().getBrowser()) {
```
Seems it's not so good solution - in any case when you need to get some properties - you need to create a new object of PropertyValue class. Let's do some googling and find a better way

PavelAnhur committed:
1. builder for Google cloud form and properties files
2. now we can get browser without creating object of class
3. added target folder in .gitignore file
